Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -184,6 +184,8 @@ namespace xr
         std::vector<Frame::Plane> Planes{};
         std::vector<Frame::Mesh> Meshes{};
         std::vector<FeaturePoint> FeaturePointCloud{};
+        std::pair<bool, Frame::Space> EyeTrackerSpace{};
```

Comment:
nit: Is the Boolean here just intended to express whether a space was detected or not? If so, might it make sense to have this be a `std::optional` instead of a pair?

Reply:
I was worried about the overhead of creating/tearing down a Frame::Space repeatedly, since we lose tracking whenever the user closes their eyes